### DISCUSSION TITLE
Push notification subscription returning 201 but no push notifications

### DISCUSSION THREAD

**Discussion by arjraj2806** [magicbell/community#204] [Questions]
Our notifications have been working fine but recently (not sure since when) all new app installs send the /push_subscription event which returns a 201 as expected, but there are no push subscriptions visible via the UI or API and no mobile_push are sent according to MagicBell logs - can someone please help?!

Additionally, even for those users where I can see mobile_push notifications in the logs - there are no registered push_subscriptions in either API or UI

**Comment by mahmoud16h** [comment]
hi @arjraj2806 did you manage to get around this at all? looks like a major issue that is not being looked at

**Comment by unamashana** [comment]
Hey @arjraj2806 and @mahmoud16h This usually means that some attribute of the push payload needs to be tweaked depending on the provider. If you are able to reproduce this reliably, I suggest passing in a provider override, or using Postman/Insomnia to try some variants of the payload. If you can let us know what helps the notification get through for specific providers, we are happy to incorporate that into our payload, or you can keep sending it as is using provider specific overrides.
